alignak/objects/config.py

```diff
@@ -2163,38 +2163,38 @@ def check_error_on_hard_unmanaged_parameters(self):
         valid = True
         if self.use_regexp_matching:
             msg = "use_regexp_matching parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.add_warning(msg)
             valid &= False
         if getattr(self, 'obsess_over_hosts', None):
             msg = "obsess_over_hosts parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.add_warning(msg)
             valid &= False
         if getattr(self, 'ochp_command', None):
             msg = "ochp_command parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.add_warning(msg)
             valid &= False
         if getattr(self, 'ochp_timeout', None):
             msg = "ochp_timeout parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.add_warning(msg)
             valid &= False
         if getattr(self, 'obsess_over_services', None):
             msg = "obsess_over_services parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.add_warning(msg)
             valid &= False
         if getattr(self, 'ocsp_command', None):
             msg = "ocsp_command parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.add_warning(msg)
             valid &= False
         if getattr(self, 'ocsp_timeout', None):
             msg = "ocsp_timeout parameter is not managed."
-            logger.error(msg)
-            self.configuration_errors.append(msg)
+            logger.warning(msg)
+            self.configuration_warnings.append(msg)
```

Comment:
couple lines before you use `self.add_warning(msg)` and here `self.configuration_warnings.append(msg)`

I could be nice to use always same code :D

Reply:
Sure it could ... but there is so many code that do not use the `add_warning` function that I prefered not to change it all. A huge *config.py* refactoring would be necessary ...